### DISCUSSION TITLE
How to attach the Goland debugger to air?

### DISCUSSION THREAD

**Discussion by AndrewGleason** [air-verse/air#446] [Q&A]
After running air, I've tried attaching to a process, only for my app to crash once I try attaching it to air.  I was curious how others use the Goland debugger with air?

**Comment by creotip** [comment]
- Add environment variables to `run configuration in your IDE.
  - in GoLand go to `Run -> Edit Configurations...`
  - Add new **_Shell Script_** configuration
  - Add new name for your configuration (for example: `air live reload`)
  - Add the following command to **_Script text_** field: `air -c .air.toml`
  - Add environment variables to **_Environment variables_** field. You can find the list of environment variables in `.env` file.
  - Run the configuration you just created.

> From now on, you can run your application with live reload by running the configuration you just created.
> Every time you change the code, the server will be rebuilt and restarted.
